qiita_db/archive.py

```diff
@@ -83,10 +85,9 @@ def get_merging_scheme_from_job(cls, job):
 
         Parameters
         ----------
-        job : qiita_db.artifact.Artifact
-            The artifact from which the features were generated
-        features : dict {str: str}
-            A dictionary of the features and the values to be stored
+        job : str
+            The Qiita process job_id generating the artifact holding the
```

Comment:
Looks like this job argument is not the ID but the `Job` object itself, right?

Reply:
yup